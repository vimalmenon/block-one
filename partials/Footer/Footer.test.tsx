import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Footer } from './'

describe('Footer Partials', () => {
  test('Footer : Snapshot testing', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
