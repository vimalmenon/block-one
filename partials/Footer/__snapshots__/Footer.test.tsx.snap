// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Partials Footer : Snapshot testing 1`] = `
<footer
  className="text-xs p-3 text-blue-700"
>
  Fictitious Bank (FB) Â© 2021 All Rights Reserved
</footer>
`;
